torch/nn/modules/module.py

```diff
@@ -2161,7 +2161,7 @@ def _named_members(self, get_members_fn, prefix='', recurse=True, remove_duplica
         for module_prefix, module in modules:
             members = get_members_fn(module)
             for k, v in members:
-                if v is None or v in memo:
+                if v is None or any(v is m for m in memo):
                     continue
                 if remove_duplicate:
                     memo.add(v)
```

Comment:
Is this also problematic? This makes using tensors with collections nearly impossible.

Reply:
I believe it is problematic. Before adding, we check `__contains__`, or something similar.

Reply:
So we can only append to list.